zerver/tornado/event_queue.py

```diff
@@ -746,6 +746,8 @@ def process_message_event(event_template, users):
             message_dict = message_dict.copy()
             message_dict["invite_only_stream"] = True
 
+        if flags is not None:
+            message_dict['is_mentioned'] = 'mentioned' in flags
         user_event = dict(type='message', message=message_dict, flags=flags) # type: Dict[str, Any]
```

Comment:
Just to make sure I understand this change, this is just a convenience to make it easier/simpler to tell whether the user is mentioned, since the following line includes `flags` in what is sent to the user, right?